packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -113,7 +122,15 @@ export const AuthContextProvider = ({
       isFirstVisit: user?.isFirstVisit ?? false,
       trackingId: user?.id,
       shouldShowLogin: loginState !== null,
-      showLogin: (trigger) => setLoginState({ trigger }),
+      showLogin: (trigger) => {
+        const hasCompanion = !!document.querySelector('daily-companion-app');
+        if (hasCompanion || authVersion === AuthVersion.V3) {
+          const signup = `${process.env.NEXT_PUBLIC_WEBAPP_URL}/signup?close=true`;
```

Comment:
When you use the close one it auto-closes the page, in that case you should opt to open in new tab.

Reply:
We are using `window.open` already. That opens a new tab.
https://github.com/dailydotdev/apps/blob/915705ae6c71b007c5c89d8a08a519c59ef8f0d1/packages/shared/src/contexts/AuthContext.tsx#L129

Reply:
Ah right, forgot `_blank` is the default 🙏